src/input-schemas.json

```
@@ -0,0 +1,1028 @@
+    },
+    "investment_cost": {
+      "UoM": "CUR/MW",
+      "constraints": null,
+      "default": null,
+      "description": "Investment cost for the asset capacity.",
+      "type": "DOUBLE"
+    },
+    "investment_cost_storage_energy": {
+      "UoM": "CUR/MWh",
+      "constraints": null,
+      "default": null,
+      "description": "Investment cost for the asset energy storage capacity.",
+      "type": "DOUBLE"
+    },
+    "investment_limit": {
+      "UoM": "MWh",
+      "constraints": null,
+      "default": null,
+      "description": "Maximum capacity for the asset investment. If the initial value is null, empty, or missing, it will be no limit.",
+      "type": "DOUBLE"
+    },
+    "investment_limit_storage_energy": {
+      "UoM": "MWh",
+      "constraints": null,
+      "default": null,
+      "description": "Maximum capacity for the asset storage investment. If the initial value is null, empty, or missing, it will be no limit.",
+      "type": "DOUBLE"
+    }
+  },
+  "asset_milestone": {
+    "asset": {
+      "UoM": null,
+      "constraints": null,
+      "default": null,
+      "description": "Name of the asset. Same as the one in the `asset` table.",
+      "type": "VARCHAR"
+    },
+    "initial_storage_level": {
+      "UoM": "MWh",
+      "constraints": {
+        "minimum": 0
+      },
+      "deafult": 0,
+      "description": "The initial storage level at the beginning of the optimization. The final storage level needs to be above this initial value. If the initial value is null, empty, or missing, it will be optimized using a cycling constraint that links the last period to the initial period.",
+      "type": "DOUBLE"
+    },
+    "investable": {
+      "UoM": null,
+      "constraints": null,
+      "default": false,
+      "description": "Whether there is an investment variable created for the asset or not.",
+      "type": "BOOLEAN"
+    },
+    "max_energy_timeframe_partition": {
+      "UoM": "MWh",
+      "constraints": {
+        "minimum": 0
+      },
+      "deafult": null,
+      "description": "The maximum amount of energy across the timeframe (e.g., a year) that the asset must produce. If the initial value is null, empty, or missing, it will be no limit.",
+      "type": "DOUBLE"
+    },
+    "milestone_year": {
+      "UoM": null,
+      "constraints": null,
+      "default": 2000,
+      "description": "Year of investment and operation decisions in the optimization.",
+      "type": "INTEGER"
+    },
+    "min_energy_timeframe_partition": {
+      "UoM": "MWh",
+      "constraints": {
+        "minimum": 0
+      },
+      "deafult": null,
+      "description": "The minimum amount of energy across the timeframe (e.g., a year) that the asset must produce. If the initial value is null, empty, or missing, it will be no limit.",
+      "type": "DOUBLE"
+    },
+    "peak_demand": {
+      "UoM": "MW",
+      "constraints": {
+        "minimum": 0
+      },
+      "deafult": 0,
```

```suggestion
      "default": 0,
```
I also noticed that there are more, and there could be more before this one, so a replace-all is adequate.